airsonic-main/src/main/java/org/airsonic/player/service/VersionService.java

```diff
@@ -188,23 +185,12 @@ public boolean isNewBetaVersionAvailable() {
      * @param resourceName The resource name.
      * @return The first line of the resource.
      */
-    private String readLineFromResource(String resourceName) {
-        InputStream in = VersionService.class.getResourceAsStream(resourceName);
-        if (in == null) {
+    private String readLineFromResource(@NonNull String resourceName) {
+        try (BufferedReader br = new BufferedReader(new FileReader(resourceName))) {
```

Comment:
Does that do the same thing as `getResourceAsStream`?

Reply:
I think it does, yes, but I prefer this construction.

Reply:
The reason I ask is that it looks like `FileReader` is expecting a path on a real filesystem, but `resourceName` could be a path to a resource inside the .war file.